pacai/util/probability.py

```diff
@@ -4,27 +4,23 @@
 
 import random
 
-from pacai.util import counter
-
-def normalize(vectorOrCounter):
+def normalize(vectorOrDict):
```

Comment:
Call it `listOrDict`.

Reply:
This is still not resolved.
Why vector over list?
Although "vector" is not technically incorrect, it is not pythonic.

Reply:
I've made the appropriate changes for the issues I've marked resolved but I haven't committed them to my repository yet.